lib/schemas/GenerateKeyOperation.js

```diff
@@ -28,6 +28,7 @@ module.exports = {
         }
       }
     },
+    // FIXME: move to keystore creation; do not provide per key
```

Comment:
Heh, let's open an issue for this instead?

Reply:
Opened #14 -- but left FIXME there to reference it so when we get to that issue we fix it.